src/config/SigInfoSchema.ts

```diff
@@ -1,5 +1,10 @@
 export interface ContributorSchema {
   githubName: string;
+  permissions?: {
```

Comment:
What contributor does not have permission?

Reply:
Because `SigInfoSchema` use the it.

Reply:
I mean the `?` suitable?